src/data/guilty_gear_xrd_rev_2/venom.yml

```diff
@@ -0,0 +1,109 @@
+---
+name: Venom
+moves:
+  normals:
+    - 5P
+    - 5K
+    - c.S
+    - f.S
+    - 5H
+    - 5D
+    - 2P
+    - 2K
+    - 2S
+    - 2H
+    - 2D
+    - 6P
+    - 6H
+    - j.P
+    - j.K
+    - j.S
+    - j.H
+    - j.D
+  universals:
+    - name: Ground Throw
+      inputs:
+        - 4H
+        - 6H
+    - name: Air Throw
+      inputs:
+        - j.4H
+        - j.6H
+    - name: Dead Angle
+      inputs:
+        - 6P+K
+      conditions: blockstunned
+    - name: Blitz
+      inputs:
+        - P+H
+        - K+H
+  specials:
+    - name: Ball Set # so. many. options.
+      inputs:
+        - '214P'
+        - '214[P]'
+        - '214K'
+        - '214[K]'
+        - '214S'
+        - '214[S]'
+        - '214H'
+        - '214[H]'
+        - 'j.214P'
+        - 'j.214[P]'
+        - 'j.214K'
+        - 'j.214[K]'
+        - 'j.214S'
+        - 'j.214[S]'
+        - 'j.214H'
+        - 'j.214[H]'
+    - name: QV
+      inputs:
+        - 41236P
+        - 41236K
+        - 41236S
+        - 41236H
+    - name: Teleport
+      inputs:
+        - 623K
+    - name: Stinger Aim
+      inputs:
+        - '[4]6S'
+        - '[4]6[S]'
+        - '[4]6H'
+        - '[4]6[H]'
+        - '[1]6S'
```

Comment:
do we need to iterate all the [1]6 combinations? does it do something different? i can't tell from dustloop